alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/AuthenticationType.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.persistence.model;
+
+import com.synopsys.integration.alert.common.rest.model.AlertSerializableModel;
+
+public class AuthenticationType extends AlertSerializableModel {
```

Comment:
Why not make this an enum and then let the accessor worry about converting to/from Long?

Reply:
Use an enum now to handle the mapping of the database id to the enumeration.  The accessor uses methods on the enum to be able to get the correct type.